python/hsfs/core/feature_monitoring_config_engine.py

```diff
@@ -408,33 +421,26 @@ def run_single_window_monitoring(
 
         return descriptive_stats
 
-    def fetch_entity_data_based_on_monitoring_window_config(
+    def fetch_entity_data_based_on_time_window_and_row_percentage(
         self,
         entity,  #: Union[feature_group.FeatureGroup, feature_view.FeatureView],
         feature_name: str,
-        monitoring_window_config: FeatureMonitoringWindowConfig,
-    ) -> Tuple[pd.DataFrame, int, int]:
-        """Fetch the entity data based on the monitoring window config.
+        start_time: int,
+        end_time: int,
+        row_percentage: int = 100,
+    ) -> pd.DataFrame:
```

Comment:
This function is used by pyspark job, that should not be a pandas dataframe.

Reply:
Fixed.